overlay/sbin/k3os-upgrade-kernel

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ $(whoami) != "root" ]
+then
+	echo "This script must be run as root."
+	exit 1
+fi
+
+PROC=$(uname -m)
+
+# Take a parameter of the version number (i.e. v0.4.0) if it is given, otherwise use latest
+if [ -z $K3OS_VERSION ]
+then
+	K3OS_VERSION=$(curl -sL api.github.com/repos/rancher/k3os/releases/latest | jq .tag_name -r)
+fi
+
+if [ $PROC == "x86_64" ]
+then
+	ARCH="amd64"
+elif [ $PROC == "aarch64" ]
+then
+	ARCH="arm64"
+elif [ $PROC == "armv7l" ]
+then
+	ARCH="arm"
```

Comment:
From what I can tell, there is only a rootfs distribution for `arm` so this script will probably "crash" if run on an arm board. Either we remove this check or use it to inform users about `arm` being a rootfs only distribution.

Reply:
Ah, that makes sense. Will adjust here to exit on arm.